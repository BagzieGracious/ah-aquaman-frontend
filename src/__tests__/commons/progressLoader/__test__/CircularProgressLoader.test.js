import { shallow } from "enzyme";
import React from "react";
import CircularProgressLoader from "../../../../commons/progressLoader";

describe("progress loader view component", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<CircularProgressLoader loading={false} />);
    expect(wrapper).toMatchSnapshot();
  });
});
