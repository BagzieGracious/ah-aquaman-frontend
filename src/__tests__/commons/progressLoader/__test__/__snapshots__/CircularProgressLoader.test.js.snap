// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`progress loader view component should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CircularProgressLoader
    loading={false}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): null,
  Symbol(enzyme.__nodes__): Array [
    null,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
